package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "uifabric-react-native",
+  "name": "@fluentui-react-native/repo",
```

Comment:
Calling this out specifically, as I kinda just chose an arbritrary name. Feel free to disagree / push back if you don't like it.